src/tasks.adoc

```diff
@@ -266,7 +266,7 @@ Add this to your `.bashrc` to get tab-complete feature on bash.
 [source,bash]
 ----
 _bb_tasks() {
-    COMPREPLY+=(`bb tasks |tail -n +3 |cut -f1 -d ' '`)
+    COMPREPLY=( $(compgen -W "$(bb-util tasks |tail -n +3 |cut -f1 -d ' ')" -- ${COMP_WORDS[COMP_CWORD]}) );
```

Comment:
what is bb-util?

Reply:
oops :)

Reply:
bb-util is a wrapper script (for running bb.edns in other directories).  Sorry about that.